src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -808,18 +808,23 @@ internal DefinitionResult GetDefinition(TextDocumentPosition textDocumentPositio
             {
                 try
                 {
+                    // Handle tokenizing for Peek Definition 
+                    Position tokenPosition = new Position();
+                    tokenPosition.Line = textDocumentPosition.Position.Line + 1;
+                    tokenPosition.Character = textDocumentPosition.Position.Character + 1;
+                    
                     // Queue the task with the binding queue
                     QueueItem queueItem = this.BindingQueue.QueueBindingOperation(
                         key: scriptParseInfo.ConnectionKey,
                         bindingTimeout: LanguageService.PeekDefinitionTimeout,
                         bindOperation: (bindingContext, cancelToken) =>
                         {
-                            string schemaName = this.GetSchemaName(scriptParseInfo, textDocumentPosition.Position, scriptFile);
+                            string schemaName = this.GetSchemaName(scriptParseInfo, tokenPosition, scriptFile);
                             // Script object using SMO
                             PeekDefinition peekDefinition = new PeekDefinition(bindingContext.ServerConnection, connInfo);
                             return peekDefinition.GetScript(
                                 scriptParseInfo.ParseResult, 
-                                textDocumentPosition.Position, 
+                                tokenPosition, 
```

Comment:
Isn't this an error? `PeekDefinition.GetScript` already adjusts line & character by 1. You need to alter that method to no longer do so.